snap/gadget.go

```diff
@@ -33,6 +34,8 @@ type GadgetInfo struct {
 
 	// Default configuration for snaps (snap-id => key => value).
 	Defaults map[string]map[string]interface{} `yaml:"defaults,omitempty"`
+
+	Connects []GadgetConnect `yaml:"connect"`
```

Comment:
This should probably be Connections/connections, to agree with the other terms (plural nouns).